packages/react-composites/src/composites/ChatComposite/Strings.tsx

```diff
@@ -13,4 +13,8 @@ export interface ChatCompositeStrings {
    * Chat list header text
    */
   chatListHeader: string;
+  /**
+   * Upload file button aria label
+   */
+  uploadFileButton: string;
```

Comment:
Does this need to be conditionally compiled?

Reply:
done